cmd/pebble/main.go

```diff
@@ -62,3 +80,13 @@ func main() {
 		muxHandler)
 	cmd.FailOnError(err, "Calling ListenAndServeTLS()")
 }
+
+func setupCustomDNSResolver(dnsResolverAddress string) {
+	net.DefaultResolver = &net.Resolver{
+		PreferGo: true,
+		Dial: func(ctx context.Context, network, address string) (net.Conn, error) {
```

Comment:
nit: I think the `network` parameter can be replaced with `_` since you aren't using it in the `Dial` func.